examples/plot_neighbors_functional_regression.py

```diff
@@ -11,20 +11,21 @@
 # sphinx_gallery_thumbnail_number = 4
 
 from sklearn.model_selection import train_test_split
-
 import matplotlib.pyplot as plt
+
 import skfda
-from skfda.ml.regression import KNeighborsFunctionalRegressor
+from skfda.ml.regression import KNeighborsRegressor
 from skfda.representation.basis import Fourier
 
 
 ##############################################################################
 #
 # In this example we are going to show the usage of the nearest neighbors
 # regressors with functional response. There is available a K-nn version,
-# :class:`~skfda.ml.regression.KNeighborsFunctionalRegressor`, and other one
-# based in the radius,
-# :class:`~skfda.ml.regression.RadiusNeighborsFunctionalRegressor`.
+# :class:`KNeighborsRegressor
+# <skfda.ml.regression.KNeighborsRegressor>`, and other one based in
```

Comment:
Use the abbreviated notation (`~path.to.class`) unless you do not want to use the name of the class in the link.

Reply:
Done in 07083ef.